lib/services/config/config.dart

```diff
@@ -362,6 +366,7 @@ class PluginConfig {
       launchMode: json['launch_mode'] == 'webview'
           ? PluginLaunchMode.webview
           : PluginLaunchMode.external,
+      action: json['action'],
```

Comment:
does this take into account a potentially `null` action?

We don't need a null check?

Reply:
if the `action` attribute in undefined in the communities JSON file, then `action` takes a null value
<img width="350" alt="Screenshot 2024-09-20 at 16 36 36" src="https://github.com/user-attachments/assets/ec4bc343-ad39-4f10-93f8-8ea093f02a07">